lib/rubocop/cop/metrics/utils/abc_size_calculator.rb

```diff
@@ -44,7 +44,10 @@ def calculate
               end
             end
 
-            Math.sqrt(@assignment**2 + @branch**2 + @condition**2).round(2)
+            [
+              Math.sqrt(@assignment**2 + @branch**2 + @condition**2).round(2),
+              "<#{@assignment},#{@branch},#{@condition}>"
```

Comment:
Might be a good idea to add a space after each `,`.

Reply:
Yep, agreed. Updated.